### DISCUSSION TITLE
Workstation GPU support

### DISCUSSION THREAD

**Discussion by Ibtisam-Mohammad** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#49] [Q&A] [answered]
Can it be used on GPU's like T4, A10, A100 etc or is there any plan for that ?
Thank you

**Comment by contentis** [comment, marked as answer]
Given there is enough VRAM, it should work on any Nvidia GPU (after Kepler). Performance gains though are most noticeable for GPUs with TensorCores aka Turing onwards. 

This being said, I only tested it on GeForce, Quadro and Mobile. If you run into any issues feel free to report them here and we can probably figure something out.

**Reply by Ibtisam-Mohammad** [reply]
Thank you for your prompt response, I shall test it on them

**Comment by arjami25** [comment]
If you're exploring options like T4, A10, or A100, you might also want to check out this guide on the [best workstation GPU](https://grafickekarty.sk/en/gpu-guides/best-workstation-gpu/) options currently available, helpful breakdown for different workloads and budgets.
